Makefile

```diff
@@ -0,0 +1,165 @@
+DOCKER_COMPOSE?=docker-compose
+EXEC?=$(DOCKER_COMPOSE) exec app
+CONSOLE=bin/console
+PHPCSFIXER?=$(EXEC) php -d memory_limit=1024m vendor/bin/php-cs-fixer
+
+.DEFAULT_GOAL := help
+.PHONY: help start stop restart install uninstall reset clear-cache tty clear clean
+.PHONY: db-diff db-migrate db-rollback db-reset db-validate wait-for-db
+.PHONY: watch assets assets-build
+.PHONY: tests lint lint-symfony lint-yaml lint-twig lint-twig php-cs php-cs-fix security-check test-schema test-all
+.PHONY: build up perm
+.PHONY: docker-compose.override.yml
+
+help:
+	@grep -E '(^[a-zA-Z_-]+:.*?##.*$$)|(^##)' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}' | sed -e 's/\[32m##/[33m/'
+
+##
+## Project setup
+##---------------------------------------------------------------------------
+
+start:                                                                                                 ## Start docker containers
+	$(DOCKER_COMPOSE) start
+
+stop:                                                                                                  ## Stop docker containers
+	$(DOCKER_COMPOSE) stop
+
+restart:                                                                                               ## Restart docker containers
+	$(DOCKER_COMPOSE) restart
+
+install: docker-compose.override.yml build up vendor perm                                                ## Create and start docker containers
+
+uninstall: stop                                                                                        ## Remove docker containers
+	$(DOCKER_COMPOSE) rm -vf
+
+reset: uninstall install                                                                               ## Remove and re-create docker containers
+
+clear-cache: perm
+	$(EXEC) $(CONSOLE) cache:clear --no-warmup
+	$(EXEC) $(CONSOLE) cache:warmup
+
+tty:                                                                                                   ## Run app container in interactive mode
```

Comment:
This part open a terminal in the container, then I've named it tty:
https://en.wikipedia.org/wiki/Computer_terminal#Text_terminals

I think sh is not really correct because there is a difference between /bin/sh and /bin/bash

Reply:
So, maybe a `bash` would be more appropriate name? BTW, I think you forgot to add a `-it` flags to allocate tty.